### PR TITLE
Using random name for PVC during tests

### PR DESCRIPTION
**Pull request by ricardomaraschini**
Allows for multiple consecutive executions of the same test.

### CODE REVIEW

File: test/e2e/pvc_test.go [hunk 1]
```diff
@@ -189,7 +191,7 @@ func TestDefaultPVC(t *testing.T) {
 }
 
 func TestCustomRWXPVC(t *testing.T) {
-	claimName := "test-custom-rwx-pvc"
+	claimName := uuid.New().String()
```
Comment (dmage) [inline test/e2e/pvc_test.go:194]: Add some prefix so we can know where it came from in case of problems.

Comment (ricardomaraschini) [reply test/e2e/pvc_test.go]: done

**Pull request merged**